src/nano-network-api.js

```diff
@@ -349,6 +349,12 @@ export class NanoNetworkApi {
         };
     }
 
+    async requestTransactionReceipts(address) {
```

Comment:
Could the NanoApi's `_requestTransactionHistory` also use this function internally? It would require a switch case over whether tha `address` param is already a `Nimiq.Address` or a string, and I don't know if that's worth it.

Reply:
I thought about that for a second, but thought it's not worth it, also with the fact in mind that NanoAPI will vanish.